src/app/modules/item/services/item-datasource.service.ts

```diff
@@ -49,7 +49,11 @@ export class ItemDataSource implements OnDestroy {
   /* state to put outputted */
   readonly state$ = this.fetchOperation$.pipe(
     delayWhen(() => this.profileLanguageMatchesAppLanguage$),
-    switchMap(item => this.fetchItemData(item).pipe(mapToFetchState({ resetter: this.refresh$ }))),
+    switchMap(item =>
+      this.userSessionService.watchedGroup$.pipe(
+        switchMap(watchedGroup => this.fetchItemData(item, watchedGroup?.route.id).pipe(mapToFetchState({ resetter: this.refresh$ })))
```

Comment:
I think using a `combineLatest` at the top would be cleaner than nesting switch maps + `.pipe(…)`:
 ```ts
combineLatest([ this.fetchOperation$, this.userSessionService.watchedGroup$ ]).pipe(
  delayWhen(() => …),
  switchMap(([ item, watchedGroup ]) => …),
)
```

Reply:
Fixed